src/vue.js

```diff
@@ -107,7 +109,9 @@ export default {
         setFaceROIAreaOffset,
         setFaceROIAreaOffsetColor,
         setFaceContours,
-        setFaceContoursColor
+        setFaceContoursColor,
+        setComputerVision,
+        setComputerVisionLoadModels
```

Comment:
These functions signatures must be in the `src/index.d.ts` too.

Reply:
Done